lib/components/admin/print-field-trip-layout.js

```diff
@@ -0,0 +1,235 @@
+      timeStamp
+    } = request
+
+    // Outbound/inbound template
+    const tripStructure = [
+      {
+        title: 'Outbound Trip (to Destination)',
+        trip: getTripFromRequest(request, true),
+        tripAbsentMessage: 'No Outbound Trip Planned'
+      },
+      {
+        title: 'Inbound Trip (from Destination)',
+        trip: getTripFromRequest(request, false),
+        tripAbsentMessage: 'No Inbound Trip Planned'
+      }
+    ]
+
+    return (
+      <PrintLayout>
+        <Header>
+          <Button bsSize='small' onClick={this._print} style={{ float: 'right' }}>
+            <i className='fa fa-print' /> Print
+          </Button>
+          <TripTitle>Field Trip Plan: {schoolName} to {endLocation}</TripTitle>
+        </Header>
+        <TripInfoList>
+          <li><b>Teacher</b>: <Val>{teacherName}</Val> ({schoolName}, Grade: <Val>{grade}</Val>)</li>
+          <li><b>Teacher Address</b>: <Val>{address}</Val></li>
+          <li><b>Phone</b>: <Val>{phoneNumber}</Val> / <b>Fax</b>: <Val>{faxNumber}</Val></li>
+          <li><b>Email</b>: <Val>{emailAddress}</Val></li>
+          <li><b>Students Age 7 and Over</b>: {numStudents || 0}</li>
+          <li><b>Students Age 6 and Under</b>: {numFreeStudents || 0}</li>
+          <li><b>Chaperones</b>: {numChaperones || 0}</li>
+          {classpassId && <li><b>Class Pass Hop Card #</b>: {classpassId}</li>}
+          <li><i>Request submitted: {timeStamp}</i></li>
+        </TripInfoList>
+
+        {tripStructure.map(({ title, trip, tripAbsentMessage }) => (
+          <TripContainer>
+            <h2>{title}</h2>
+            {trip
+              ? trip.groupItineraries?.map((groupItin, i) => {
+                const itinerary = JSON.parse(lzwDecode(groupItin.itinData))
+                return (
+                  <TripBody key={i}>
+                    <ItineraryContainer>
+                      <h3>{groupItin.passengers} passengers on following itinerary:</h3>
+                      <ItineraryBody>
+                        <PrintableItinerary
+                          config={config}
+                          itinerary={itinerary}
+                          LegIcon={LegIcon}
+                        />
+                        <TripSummary itinerary={itinerary} />
+                      </ItineraryBody>
+                    </ItineraryContainer>
+                  </TripBody>
+                )
+              })
+              : <TripBody><i>{tripAbsentMessage}</i></TripBody>
+            }
+          </TripContainer>
+        ))}
+      </PrintLayout>
+    )
+  }
+}
+
+// connect to the redux store
+
+const mapStateToProps = (state, ownProps) => {
+  const requestId = parseInt(state.router.location.query.requestId)
+  const { requests } = state.callTaker.fieldTrip
+  const request = requests.data.find(req => req.id === requestId)
+  return {
+    config: state.otp.config,
+    request,
+    requestId,
+    session: state.callTaker.session,
+    title: getTitle(state)
```

Comment:
I think a better title might be the `Field Trip: {schoolName} to {endLocation}` string, but this is non-blocking and we should just get TriMet input on this at a later point.

Reply:
Sure, updated in https://github.com/opentripplanner/otp-react-redux/pull/398/commits/e5ace268f1d2f470a533eae0e70be3916c4076d0.